src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26066.cs

```diff
@@ -6,18 +6,24 @@ namespace Microsoft.Maui.TestCases.Tests.Issues;
 
 public class Issue26066(TestDevice testDevice) : _IssuesUITest(testDevice)
 {
+    const string CV2Item1 = "CV2-Item1";
+
+#if MACCATALYST
+    const string Ok = "action-button--999";
```

Comment:
Can this be generalized somewhere when we need to click on the OK button of a displayalert?

Do we know why OK doesn't work on Catalyst? It seems like the automation identifier should be set to OK on Catalyst and reachable as well

Reply:
@PureWeen, Upon investigation, the display alert 'OK' button's automation identifier isn't set to the button text as we expected. Instead, it's using a system-generated ID: 'action-button--999'.
 
Added helper method to tap display alert action button and committed the changes, kindly review and share your concerns.

![image](https://github.com/user-attachments/assets/3d41a788-318f-46ab-be55-b4295cde9c2b)